_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -11,17 +11,191 @@ redirect_from:
+    <tr>
+        <td>Test execution time (minutes)</td>
+        <td></td>
+        <td>40.00</td>
+        <td>22.00</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td rowspan="3">Throughput (number of requests per second)</td>
+        <td>p0</td>
+        <td>17553.90</td>
+        <td>28584.30</td>
+        <td>63%</td>
+    </tr>
+    <tr>
+        <td>p50</td>
+        <td>20647.20</td>
+        <td>32790.20</td>
+        <td>59%</td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td>23209.00</td>
+        <td>34286.00</td>
+        <td>48%</td>
+    </tr>
+    <tr>
+        <td rowspan="4">CPU (%)</td>
+        <td>p50</td>
+        <td>65.00</td>
+        <td>30.00</td>
+        <td>-54%</td>
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td>79.00</td>
+        <td>35.00</td>
+        <td>-56%</td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td>98.00</td>
+        <td>45.08</td>
+        <td>-54%</td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td>98.00</td>
+        <td>59.00</td>
+        <td>-40%</td>
+    </tr>
+    <tr>
+        <td rowspan="4">Memory (%)</td>
+        <td>p50</td>
+        <td>48.20</td>
+        <td>39.00</td>
+        <td>-19%</td>
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td>62.00</td>
+        <td>61.00</td>
+        <td>-2%</td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td>66.21</td>
+        <td>68.00</td>
+        <td>3%</td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td>71.00</td>
+        <td>69.00</td>
+        <td>-3%</td>
+    </tr>
+    <tr>
+        <td rowspan="4">IOPS</td>
+        <td>p50</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td rowspan="4">Latency</td>
+        <td>p50</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p90</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p99</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>p100</td>
+        <td></td>
+        <td></td>
+        <td></td>
+    </tr>
+</table>
+
+**Note** : Your results may vary based on the cluster topology, hardware used, shard count, and merge settings. 
+
+Your results may vary based on cluster topology, hardware used, shard count, and merge settings. 
+{: .note }
+
+## Further resources regarding segment replication
+
+1. Track [future enhancements on segment replication](https://github.com/orgs/opensearch-project/projects/99).
+2. [Segment replication blog post](https://github.com/opensearch-project/project-website/pull/1504/files).
+
+## Other considerations
```

Comment:
We should introduce this section. 

"When using segment replication, consider the following best practices:"